tests/flow/test_prev_rdb_decode.py

```
@@ -56,6 +59,10 @@ def generate_dump(key, port):
 
     # Select the social graph
     g = db.select_graph(key)
+    try:
+        g.delete()
+    except:
+        pass
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(BaseException)`

(SIM105)